akka-stream/src/main/scala/akka/stream/extra/Implicits.scala

```diff
@@ -18,19 +18,19 @@ object Implicits {
    *
    * See [[Timed]]
    */
-  implicit class TimedSourceDsl[I](val flow: Source[I]) extends AnyVal {
+  implicit class TimedSourceDsl[I, Mat](val source: Source[I, Mat]) extends AnyVal {
 
     /**
      * Measures time from receieving the first element and completion events - one for each subscriber of this `Flow`.
      */
-    def timed[O](measuredOps: Source[I] ⇒ Source[O], onComplete: FiniteDuration ⇒ Unit): Source[O] =
-      Timed.timed[I, O](flow, measuredOps, onComplete)
+    def timed[O, Mat2](measuredOps: Source[I, Mat] ⇒ Source[O, Mat2], onComplete: FiniteDuration ⇒ Unit): Source[O, Mat2] =
```

Comment:
another case of Source => Source pattern which just remained from the time when we only had Flow which was actually a source.